src/App.tsx

```diff
@@ -1,61 +1,218 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+
+export type User = (typeof usersFromServer)[0];
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  createdAt: Date;
+}
+
+export interface UserTodos {
+  userId: number;
+  userName: string;
+  userEmail: string;
+  userLogin: string;
+  todoPosts: Todo[];
+}
+
+const usersTodos = usersFromServer
+  .map(user => {
+    const userTodo = todosFromServer.reduce((todosAcc: Todo[], todo) => {
+      if (user.id === todo.userId) {
+        todosAcc.push({ ...todo, ...{ createdAt: new Date() } });
+
+        return todosAcc;
+      }
+
+      return todosAcc;
+    }, []);
+
+    return {
+      userId: user.id,
+      userName: user.name,
+      userEmail: user.email,
+      userLogin: user.username,
+      todoPosts: userTodo,
+    };
+  })
+  .filter(userTodo => userTodo.todoPosts.length > 0);
+
+function findTheBiggestPostId(postList: UserTodos[]): number {
+  return Math.max(
+    ...postList.map(userPost =>
+      Math.max(...userPost.todoPosts.map(post => post.id)),
+    ),
+  );
+}
 
 export const App = () => {
+  // #region baseStates
+  const [currentUsersTodos, setCurrentUsersTodos] =
+    useState<UserTodos[]>(usersTodos);
+  const [postCount, setPostCount] = useState(
+    findTheBiggestPostId(currentUsersTodos) + 1,
+  );
+  // #endregion
+  // #region title
```

Comment:
Remove all comments

Reply:
it's for more comfortable code view and fast orientation in my code, it's just for convenience.